pages/phases/job-analysis/defining-competencies.md

```diff
@@ -6,7 +6,7 @@ title: Defining Competencies
 sidenav: hiring-phases
 phase: job-analysis
 sub-phase-order: 040
-intro: Resume reviewers and interviewers must be able to clearly and objectively evaluate an applicant against competencies that you've defined before assessments begin. To define your competencies, document what it looks like when an applicant has that skill. You'll list these definitions in the job announcement as required qualifications.
+intro: During the job analysis workshop, SMEs will clearly define the specialized experience in the form of competencies. These will then be listed in the job announcement along with the competency definition. 
```

Comment:
i think i made this active instead of passive.

Reply:
alas, I think the reverse is the case. Editing.